### PR TITLE
Update Langauage Settings File

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update Language Settings File

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -106,7 +106,8 @@ function Publish-dotnet-GithubIODocs ()
       Write-Host "DocDir $($Item)"
       Write-Host "PkgName $($PkgName)"
       Write-Host "DocVersion $($DocVersion)"
-      Upload-Blobs -DocDir "$($Item)" -PkgName $PkgName -DocVersion $DocVersion
+      $releaseTag = RetrieveReleaseTag "Nuget" $PublicArtifactLocation
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:109]: There are a lot of variables being accessed in this function that aren't passed in. How do we ensure everything is setup correctly before we call this?

Comment (chidozieononiwu) [reply eng/scripts/Language-Settings.ps1:109]: Will have to import `eng\common\scripts\artifact-metadata-parsing.ps1` in `common.ps1` for this to work. That should work. Bit Ill have it tested when submitting the tools PR.

Comment (weshaggard) [reply eng/scripts/Language-Settings.ps1:109]: If you are trying to keep things all updated you should also update PackageInfoFromPackageFile to add the ReleaseTag

Comment (chidozieononiwu) [reply eng/scripts/Language-Settings.ps1:109]: Am not calling the `PackageInfoFromPackageFile ` in this file just yet.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**